packages/react-components/src/lib/components/NetworkDropdown.tsx

```diff
@@ -28,13 +27,13 @@ const iconForNetwork = (network: NetworkConfig) => {
 };
 
 export const NetworkDropdown = ({
-  networkConfigs,
   label,
   maxHeight,
   size = 'md',
   appearance = 'bold',
 }: NetworkDropdownProps) => {
-  const { networkConfig, setNetworkConfig } = useAgoricNetwork();
+  const { networkConfig, setNetworkConfig, agoricNetworkConfigs } =
```

Comment:
why is this component mixing `agoricNetwork` from chain-registry and this `useAgoricNetwork` hook? I'd expect `useAgoricNetwork` to abstract all network selection.

Same for `nameForNetwork` and `iconForNetwork`.

Is that reasonable?